filament/src/Renderer.cpp

```diff
@@ -258,43 +258,44 @@ void FRenderer::renderJob(ArenaScope& arena, FView& view) {
 
     FrameGraph fg(engine.getResourceAllocator());
 
-    const TargetBufferFlags discardedFlags = view.getDiscardedTargetBuffers();
+    const TargetBufferFlags discardedFlags = mDiscardedFlags;
+    const TargetBufferFlags clearFlags = mClearFlags;
+    const float4 clearColor = mClearOptions.clearColor;
 
-    TargetBufferFlags clearFlags = (view.getClearFlags() & TargetBufferFlags::COLOR);
+    // once we've done the first View, we don't discard or clear the color buffer for the next ones
```

Comment:
This comment is confusing to me.. does this mean we only clear color for the first view (if clear is enabled)?

Reply:
okay will update comment (then you can tell me if it's clearer :))